doc/languages-frameworks/cuda.section.md

```diff
@@ -54,3 +54,52 @@ for your specific card(s).
 
 Library maintainers should consult [NVCC Docs](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/)
 and release notes for their software package.
+
+## Adding a new CUDA release (#adding-a-new-cuda-release)
+
+> **WARNING**
+>
+> This section of the docs is still very much in progress. Apologies.
+
+For the CUDA Toolkit:
+
+1. Go to NVIDIA's CUDA Toolkit download page: <https://developer.nvidia.com/cuda-downloads>
+2. Select the appropriate OS, architecture, distribution, and version, and installer type.
+
+   - For example: Linux, x86_64, Ubuntu, 22.04, runfile (local)
+   - NOTE: Typically, we use the Ubuntu runfile. It is unclear if the runfile for other distributions will work.
+
+3. Take the link provided by the installer instructions on the webpage after selecting the installer type and get its hash by running:
+
+   ```bash
+   nix store prefetch-file --hash-type sha256 <link>
+   ```
+
+4. Update `pkgs/development/compilers/cudatoolkit/versions.toml` to include the release.
+
+For CUDA redistributables:
```

Comment:
Sounds as if "cuda redistributables" are opposed to "cudatoolkit" from the previous paragraph. I'd move this paragraph up and start it with "the preferred way to package a new cudatoolkit release in nixpkgs is from the 'redistributable' components published at..."

Reply:
d990c03ed934056291dc3d79d00eb52a2189968d